apps/fabric-website/src/components/App/AppState.tsx

```diff
@@ -380,6 +385,7 @@ export const AppState: IAppState = {
             {
               title: 'Themes',
               url: '#/components/themes',
+              isHiddenFromMainNav: true,
```

Comment:
I don't think we'd want to hide this right now, do we? This functionality is useful to demo, even if the generator tool isn't available.